pkgs/by-name/pa/pam_fprint_grosshack/package.nix

```
@@ -0,0 +1,61 @@
+{
+  lib,
+  stdenv,
+  fetchurl,
+  pkg-config,
+  meson,
+  ninja,
+  glib,
+  libfprint,
+  polkit,
+  systemd,
+  pam,
+  libpam-wrapper,
+  dbus,
+}:
+
+stdenv.mkDerivation rec {
+
+  pname = "pam_fprint_grosshack";
+  version = "0.3.0";
+  outputs = [ "out" ];
+
+  src = fetchurl {
+    url = "https://gitlab.com/mishakmak/pam-fprint-grosshack/-/archive/v${version}/pam-fprint-grosshack-v0.3.0.tar.gz";
+    sha256 = "sha256-5NiVV0o0dkc6SsWgKT09/zgMw8lZtWTlSUqV2qnYsV8=";
+  };
```

why can't we use fetch from gitlab?
```suggestion
  src = fetchFromGitLab {
    owner = "mishakmak";
    repo = "pam-fprint-grosshack";
    tag = "v${version}";
    sha256 = "sha256-5NiVV0o0dkc6SsWgKT09/zgMw8lZtWTlSUqV2qnYsV8=";
  };
```

---

That's a great idea, will implement it. Didn't really know about fetchFromGitlab to be honest.